lib/modules/commentTools.js

```diff
@@ -832,6 +832,17 @@ const magicPlaceholders = [
 			return date.toDateString();
 		},
 	}, {
+		matches: ['flair'],
```

Comment:
Could you change this to the more specific `linkflair`, to disambiguate from user flair?